app/src/main/kotlin/com/intive/picover/parties/repository/PartiesRepository.kt

```diff
@@ -20,6 +21,13 @@ class PartiesRepository @Inject constructor(
 			}
 		}
 
+	suspend fun addParty(partyRemote: PartyRemote) =
+		runCatching {
+			partiesReference.push().let {
+				it.setValue(partyRemote.copy(id = it.key ?: "")).await()
```

Comment:
it.key can be null ? in which situation ? is it ok that in some situation we will add a party with empty id string ?

Reply:
That also leads to another conclusion, if it's acceptable to set an id to `""` then what it such case happen more than once? Is it correct to have two objects with the same `id`?

Reply:
According to the documentation "
The last token in the location pointed to by this reference or null if this reference points to the database root". So very unlikely, but I get the point. Code updated.

Reply:
@marcinWisnia @MarekMacko Updated. I was wondering if I should throw more specific error such as IllegalStateExcepption but I think a general error message 'Something went wrong" should be enough for such a corner case.